src/validation/tests/snapshots/rusty__validation__tests__recursive_validation_tests__arrays__one_cycle_aba_input.snap

```diff
@@ -6,7 +6,10 @@ error: Recursive data structure `A -> B -> A` has infinite size
   ┌─ <internal>:2:18
   │
 2 │             TYPE A : STRUCT
-  │                  ^ Recursive data structure `A -> B -> A` has infinite size
+  │                  ^
+  │                  │
+  │                  Recursive data structure `A -> B -> A` has infinite size
+  │                  see also
```

Comment:
Something has changed for this validation, looks like the location is both present in the primary and secondary location?